README.md

```diff
@@ -13,6 +13,10 @@
 
 </div>
 
+## ❤️ Note for new Contributors : ❤️
+
+Thank you very much for considering this project. Make sure you read [Code of Conduct](https://github.com/kaiwalyakoparkar/flashtype/blob/master/CODE_OF_CONDUCT.md) and [Contribution Guidelines](https://github.com/kaiwalyakoparkar/flashtype/blob/master/CONTRIBUTING.md) before making contributions. Feel free to raise an issue and after gettting assigned to it, you are all set to work on it.
```

Comment:
Instead of the entire link. You can add CODE_OF_CONDUCT.md as a link itself. like [ code of conduct (CODE_OF_CONDUCT.md)